statement.go

```diff
@@ -169,7 +168,7 @@ func (c converter) ConvertValue(v interface{}) (driver.Value, error) {
 	case reflect.Uint64:
 		u64 := rv.Uint()
 		if u64 >= 1<<63 {
```

Comment:
Is there any reason to still do this check for the high bit instead of just always returning `u64`?